src/components/region-common.js

```diff
@@ -26,3 +26,31 @@ export const regionInfo = (location, locationType) => {
       return { flag: unknownRegionFlag, regionDescription: `${locationType}: ${location}` }
   }
 }
+export const allRegions = [
```

Comment:
We have made a product decision that for now, we want to avoid creating within-country silos of life sciences community data (for example, data in `us-east` cannot be cross-analyzed with data in `us-central` without incurring egress charges). This means that we are going to be opinionated and pick specific regions in countries that have multiple regions. This comes to do US and Japan.

Thus we should remove from this list (and add a comment here) that `us-east*`, `us-west*`, and `asia-northeast2` are unavailable. We will put into documentation that if people want these regions, they should reach out to Terra support so that we can discuss the use cases.